tests/integration-tests/tests/update/test_update.py

```diff
@@ -439,3 +441,144 @@ def get_batch_spot_bid_percentage(stack_name, region):
         .get("computeResources")
         .get("bidPercentage")
     )
+
+
+@pytest.mark.usefixtures("instance")
+def test_queue_parameters_update(
+    region, os, pcluster_config_reader, ami_copy, clusters_factory, scheduler_commands_factory, request
+):
+    """Test update cluster with drain strategy."""
+    # Create cluster with initial configuration
+    ec2 = boto3.client("ec2", region)
+    pcluster_ami_id = retrieve_latest_ami(region, os, ami_type="pcluster", request=request)
+    init_config_file = pcluster_config_reader(global_custom_ami=pcluster_ami_id)
+    cluster = clusters_factory(init_config_file)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue1")
+    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue2")
+
+    _test_queue_strategy_without_running_job(
+        pcluster_config_reader, pcluster_ami_id, cluster, region, os, remote_command_executor, scheduler_commands
+    )
+
+    queue1_job_id = scheduler_commands.submit_command_and_assert_job_accepted(
+        submit_command_args={
+            "command": "sleep 3000",
+            "nodes": -1,
+            "partition": "queue1",
+            "other_options": "-a 1-5",  # instance type has 4 cpus per node, which requre 2 nodes to run the job
+        }
+    )
+
+    queue2_job_id = scheduler_commands.submit_command_and_assert_job_accepted(
+        submit_command_args={"command": "sleep 3000", "nodes": -1, "partition": "queue2", "other_options": "-a 1-5"}
+    )
+    # Wait for the job to run
+    scheduler_commands.wait_job_running(queue1_job_id)
+    scheduler_commands.wait_job_running(queue2_job_id)
+
+    pcluster_copy_ami_id = ami_copy(
+        pcluster_ami_id, "-".join(["test", "update", "computenode", generate_random_string()])
+    )
+    # test update without setting queue strategy, update will fail
+    _test_update_without_queue_strategy(pcluster_config_reader, pcluster_ami_id, pcluster_copy_ami_id, cluster)
+
+    updated_config_file = pcluster_config_reader(
```

Comment:
let's put this in another def, e.g. _test_update_queue_strategy_with_running_job

Reply:
Changed